packages/uikit/src/icons/index.ts

```diff
@@ -0,0 +1,22 @@
+export * from "react-icons/fa";
```

Comment:
If the package is updated this approach can be a nightmare, or we change the lib.
It should look like 

```typescript
export * from 'react-icons'
// just this
```

Could you locate this file in `/Components/Icons/index.ts`?